lib/secrets.py

```diff
@@ -1,4 +1,4 @@
 class secrets_login(object):
     def __init__(self):
-        self.email = .......
-        self.password = ......
\ No newline at end of file
+        self.email = "rufina@julofinance.com"
```

Comment:
the creds is not correct. pls check on test_login.py file. this exercise using "https://reqres.in/api/login" as endpoint